### PR TITLE
Added Double D Electronics DDA75

### PR DESCRIPTION
**Pull request by robshot**

### CODE REVIEW

File: connector/doc/Double_D_Electronics_DDA75.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+- Interface connection:
+
+  - **IP address/host**: The polling IP or URL of the destination.
+  - **IP port**: The IP port of the destination.
+  - **Bus address**: The bus address of the device.
+
+## How to use
+
+On the **General** data page, you can find the status of the first four ports with each 8 bit and some generic alarm flags.
```
Comment (MariekeGO) [inline connector/doc/Double_D_Electronics_DDA75.md:47]: Could you rephrase this sentence? It isn't quite clear to me what you mean with "with each 8 bit".

**Comment by MariekeGO** [comment]
Could you add your full name to your GitHub profile so we can assign points to you if you are a member of the DevOps Program?

**Comment by robshot** [comment]
Updated my profile as well.

**Review by MariekeGO — APPROVED** [review]
